tools.py

```diff
@@ -121,14 +121,32 @@ def js_to_text(js):
     except TypeError:
         return text
 
+    def _replace_br(value):
+        return re.sub(r'<br\s*/?>', r'\\n', value, flags=re.IGNORECASE)
+
     for block in js_object['blocks']:
-        if 'text' in block['data']:
-            text += block['data']['text'].replace('<br>' , '\\n')
-        if 'items' in block['data']:
-            for entry in block['data']['items']:
-                text += '>' + str(entry).replace('<br>' , '') + '\\n'
-        text += '\\n'
-    return str(text)
+        type_ = block.get('type', 'paragraph')
+        if type_ == 'header':
+            text += '# %s\\n' % _replace_br(block['data']['text'])
+        elif type_ == 'list':
+            for item in  block['data']['items']:
+                text += '- %s\n' % _replace_br(item)
+        elif type_ == 'quote':
+            text += '> %s\\n' % _replace_br(block['data']['text'])
+        elif type_ == 'code':
+            text += 'code:\n%s\\n' % block['data']['code']
+        elif type_ == 'image':
+            image = None
+            if block['data'].get('url'):
+                text += '%s\\n' % block['data']['url']
+            elif block['data'].get('file'):
+                text += 'image: %s\\n' % block['data']['file']['url']
```

Comment:
Does this follow the syntax of markdown for images?

https://www.markdownguide.org/basic-syntax/#images